wandb/sdk/launch/runner/kubernetes.py

```diff
@@ -161,15 +183,22 @@ def run(self, launch_project: LaunchProject) -> Optional[AbstractRun]:  # noqa:
         if resource_args.get("suspend"):
             job_spec["suspend"] = resource_args.get("suspend")
 
+        pod_spec["restartPolicy"] = resource_args.get("pod_restart_policy", "Never")
+        if resource_args.get("pod_preemption_policy"):
+            pod_spec["preemptionPolicy"] = resource_args.get("pod_preemption_policy")
+        if resource_args.get("node_name"):
+            pod_spec["nodeName"] = resource_args.get("node_name")
+        if resource_args.get("node_selectors"):
+            pod_spec["nodeSelectors"] = resource_args.get("node_selectors")
+
         # only support container overrides for the single container case
         if any(
             arg in resource_args
-            for arg in ["container_name", "resource_requests", "resource_limits"]
+            for arg in ["container_name", "resource_requests", "resource_limits", "env"]
```

Comment:
Why add env here? What if the user wants to specify env vars for their container?

Reply:
oh this should have an `and` on here for the multiple container case, fixed